h2o-core/src/main/java/water/DKVManager.java

```diff
@@ -0,0 +1,9 @@
+package water;
+
+public class DKVManager {
+  
+  public void retain(final Key[] retainedKeys){
```

Comment:
Will be called via a function, this is just a helper during development right now, please ignore. The classes will look nicer in the end.